variables.tf

```diff
@@ -102,3 +102,41 @@ variable "infrastructure_vpc_network_availability_zones" {
   description = "A list of availability zone characters (eg. [\"a\", \"b\", \"c\"])"
   type        = list(string)
 }
+
+variable "infrastructure_vpc_network_acl_egress_lockdown" {
+  description = "Creates a network ACL which blocks all egress traffic, except the ports required for services that are launched."
```

Comment:
Where are 'the ports required for services that are launched' defined?

Reply:
This will be services launched when a feature is enabled, eg. an ECS cluster and loadbalancer - for which would require port 80/433 (and ALB ephemeral ports) opened
Perhaps there's a better description?

Reply:
```hcl
description = "Disable all custom network egress rules, leaving only the defaults required for the service to operate"
```